### ISSUE TITLE
`remove_label_rows()` converts dates to NA

### ISSUE THREAD

**Issue by jstevens5** [jstevens5/excluder#4]
Instead of converting the dates columns of metadata from characters to dates, `remove_label_rows()` is converting to NA with warning.
``` 
Warning messages:
1: Problem with `mutate()` input `StartDate`.
i  74 failed to parse.
i Input `StartDate` is `lubridate::ymd_hms(.data$StartDate)`. 
2: Problem with `mutate()` input `EndDate`.
i  73 failed to parse.
i Input `EndDate` is `lubridate::ymd_hms(.data$EndDate)`. 
3: Problem with `mutate()` input `RecordedDate`.
i  68 failed to parse.
i Input `RecordedDate` is `lubridate::ymd_hms(.data$RecordedDate)`. 
```

**Comment by jstevens5** [comment]
This occurs when the date is not in a YYYY-MM-DD HH:MM:SS format.

**Comment by jstevens5** [comment]
Fixed by 4a48354b7b205e7e316abb9edc2ba21ce5b9654a
